grammar.js

```diff
@@ -742,6 +742,7 @@ module.exports = grammar({
 
     /* Shared AST nodes */
     identifier: ($) => $._name,
+    label: ($) => $._name,
```

Comment:
I added this new node `(label)` that takes `(identifier)`'s place in some cases. It really simplifies the `locals.scm` queries :confetti_ball: